comp/etw/component.go

```diff
@@ -185,6 +185,7 @@ type Session interface {
 // Component offers a way to create ETW tracing sessions with a given name.
 type Component interface {
 	NewSession(sessionName string) (Session, error)
+	NewWellKnownSession(sessionName string) (Session, error)
```

Comment:
Something to contemplate with @julien-lebot, I feel that ConnectToExternalSession would be better describing semantics of this and subsequent operations. We probably should have Disconnect method for a session as well then.